catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -401,10 +401,7 @@ public static void setAutoCrashReportingEnabled(Context context, boolean isEnabl
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_CRASH_REPORTS, isEnabled);
 		editor.commit();
-
-		if (isEnabled) {
-			CrashReporter.initialize(context);
-		}
+		CrashReporter.initialize(context);
```

Comment:
Please undo as was. Calling initialize even if you don't want to initialize the CrashReporter is really really odd.